pkg/scheduler/framework/plugins/sameplacementaffinity/filtering_test.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package sameplacementaffinity
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	fleetv1beta1 "go.goms.io/fleet/apis/placement/v1beta1"
+	"go.goms.io/fleet/pkg/scheduler/framework"
+)
+
+const (
+	clusterName = "member-1"
+)
+
+var (
+	cmpStatusOptions = cmp.Options{
+		cmpopts.IgnoreFields(framework.Status{}, "reasons", "err"),
+		cmp.AllowUnexported(framework.Status{}),
+	}
+	defaultPluginName = defaultPluginOptions.name
+)
+
+func TestFilter(t *testing.T) {
+	tests := []struct {
+		name                     string
+		scheduledOrBoundBindings []*fleetv1beta1.ClusterResourceBinding
+		want                     *framework.Status
+	}{
+		{
+			name: "placement has already been scheduled",
```

Comment:
Just a nit, the name should be `placement has been unscheduled` I assume?

Reply:
(The matching binding is unscheduled)

Reply:
The binding should be scheduled. Fixed.